pepper-apis/src/main/java/org/broadinstitute/ddp/filter/WhiteListFilter.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.ddp.filter;
+
+import static java.util.stream.Collectors.toList;
+import static spark.Spark.before;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+import org.broadinstitute.ddp.util.ResponseUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+
+public class WhiteListFilter implements Filter {
+    private static final Logger LOG = LoggerFactory.getLogger(WhiteListFilter.class);
+    private Set<String> ipSet;
+
+    public static void whitelist(String path, Collection<String> ips) {
+        WhiteListFilter filter = new WhiteListFilter(ips);
+        before(path, (request, response) -> {
+            filter.handle(request, response);
+        });
```

Comment:
I don't think we need to wrap it a closure -- we can just pass `filter` as 2nd argument to `before()`?